akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Plugins.scala

```diff
@@ -96,17 +97,52 @@ object PluginModel {
     val pluginWithAbsoluteJsEntrypoint = replaceRelativeJavascriptEntrypoint(plugin)
     replaceRelativeLocalesBaseUrl(pluginWithAbsoluteJsEntrypoint)
   }
-  def createPluginModelFromJson(json: util.Map[String, AnyRef]): PluginModel = {
+
+  private def checkPluginSdkVersionDevMode(pluginSdkVersion: String): Boolean =
+    pluginSdkVersion.startsWith("http://codeload")
+
+  private def versioningComparison(v1: String, v2: String): Int = {
```

Comment:
This function is not working properly.
I set in my manifest.json `"requiredSdkVersion": "~1.0.79",` and in bbb-web.properties `SdkVersion=0.0.79`.
And it imported the plugin.

Reply:
We should avoid homemade solution for such complex comparison. I sent a PR.